patches/@artsy/stitch+6.0.0.patch

```diff
@@ -0,0 +1,283 @@
+ 
+-  var BLACKLIST = [".js", ".jsx", ".ts", ".tsx"];
+-  var found = BLACKLIST.some(function (extension) {
+-    return block.includes(extension);
+-  });
+-  return !found;
++function isTemplate(filePath) {
++  return !BLACKLIST.includes(_path.default.extname(filePath));
+ }
+ 
+ function isComponent(block) {
+--- a/node_modules/@artsy/stitch/dist/utils.js.map
++++ b/node_modules/@artsy/stitch/dist/utils.js.map
+@@ -1 +1 @@
+-{"version":3,"sources":["../src/utils.tsx"],"names":["isTemplate","block","BLACKLIST","found","some","extension","includes","isComponent","Error"],"mappings":";;;;;;;;AAAA;;AAGO,SAASA,UAAT,CAAoBC,KAApB,EAA2C;AAChD,MAAI,CAAC,sBAASA,KAAT,CAAL,EAAsB;AACpB,WAAO,KAAP;AACD;;AACD,MAAMC,SAAS,GAAG,CAAC,KAAD,EAAQ,MAAR,EAAgB,KAAhB,EAAuB,MAAvB,CAAlB;AACA,MAAMC,KAAK,GAAGD,SAAS,CAACE,IAAV,CAAe,UAAAC,SAAS;AAAA,WAAIJ,KAAK,CAACK,QAAN,CAAeD,SAAf,CAAJ;AAAA,GAAxB,CAAd;AACA,SAAO,CAACF,KAAR;AACD;;AAEM,SAASI,WAAT,CAAqBN,KAArB,EAA4C;AACjD,MAAI,wBAAWA,KAAX,CAAJ,EAAuB;AACrB,WAAO,IAAP;AACD,GAFD,MAEO;AACL,UAAM,IAAIO,KAAJ,CACJ,gCACE,4CAFE,CAAN;AAID;AACF","sourcesContent":["import { isFunction, isString } from \"lodash\"\nimport { Block } from \"./render\"\n\nexport function isTemplate(block: Block): boolean {\n  if (!isString(block)) {\n    return false\n  }\n  const BLACKLIST = [\".js\", \".jsx\", \".ts\", \".tsx\"]\n  const found = BLACKLIST.some(extension => block.includes(extension))\n  return !found\n}\n\nexport function isComponent(block: Block): boolean {\n  if (isFunction(block)) {\n    return true\n  } else {\n    throw new Error(\n      \"(@artsy/stitch: lib/utils) \" +\n        \"Error rendering layout: Invalid component.\"\n    )\n  }\n}\n"],"file":"utils.js"}
+\ No newline at end of file
++{"version":3,"sources":["../src/utils.tsx"],"names":["BLACKLIST","isTemplate","filePath","includes","path","extname","isComponent","block","Error"],"mappings":";;;;;;;;AAAA;;AACA;;;;AAGA,IAAMA,SAAS,GAAG,CAAC,KAAD,EAAQ,MAAR,EAAgB,KAAhB,EAAuB,MAAvB,CAAlB,C,CAEA;;AACO,SAASC,UAAT,CAAoBC,QAApB,EAA+C;AACpD,SAAO,CAACF,SAAS,CAACG,QAAV,CAAmBC,cAAKC,OAAL,CAAaH,QAAb,CAAnB,CAAR;AACD;;AAEM,SAASI,WAAT,CAAqBC,KAArB,EAA4C;AACjD,MAAI,wBAAWA,KAAX,CAAJ,EAAuB;AACrB,WAAO,IAAP;AACD,GAFD,MAEO;AACL,UAAM,IAAIC,KAAJ,CACJ,gCACE,4CAFE,CAAN;AAID;AACF","sourcesContent":["import { isFunction, isString } from \"lodash\"\nimport path from \"path\"\nimport { Block } from \"./render\"\n\nconst BLACKLIST = [\".js\", \".jsx\", \".ts\", \".tsx\"]\n\n// TODO: Could stat the FS to make sure this is truly an existing template.\nexport function isTemplate(filePath: string): boolean {\n  return !BLACKLIST.includes(path.extname(filePath))\n}\n\nexport function isComponent(block: Block): boolean {\n  if (isFunction(block)) {\n    return true\n  } else {\n    throw new Error(\n      \"(@artsy/stitch: lib/utils) \" +\n        \"Error rendering layout: Invalid component.\"\n    )\n  }\n}\n"],"file":"utils.js"}
+\ No newline at end of file
```

Comment:
Patch package FTW

Reply:
I know, right? I absolutely ❤️ it. This package speaks my language.